docs/docs-content/clusters/palette-virtual-clusters/configure-oidc-virtual-cluster.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_label: "Configure OIDC for a Virtual Cluster"
+title: "Configure OIDC for a Virtual Cluster"
+description: "Learn how to configure OIDC for Palette Virtual Clusters."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 5
+tags: ["clusters", "cluster groups", "virtual clusters"]
+---
+
+
+Before deploying a virtual cluster, you must set up OpenID Connect (OIDC) Identity Provider (IDP) for the cluster group the virtual cluster belongs to. As virtual clusters are deployed to that cluster group, OIDC information in the cluster group configuration will be inserted into the virtual cluster configuration. 
+
+
+## Prerequisites
+
+- A healthy host cluster that you will use to create a cluster group.
+- A cluster group to which you can add virtual clusters.
+- An issuer URL and client ID that you obtain from your identity provider.
+- Privileges sufficient to edit cluster group settings and a viewer role to access the generated kubeconfig. If you are deploying virtual clusters, you will need admin privileges.
+- [kubelogin](https://github.com/int128/kubelogin) installed. This is a kubectl plugin for Kubernetes OIDC authentication, also known as `kubectl` oidc-login.
+
+
+## Configure OIDC
+
+Use these steps to configure your cluster group with OIDC *before* creating a virtual cluster.
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. If a cluster group does not exist, you will need to create a host cluster and then create the cluster group. Refer to [Create and Manage Cluster Groups](../../clusters/cluster-groups/create-cluster-group.md) for guidance.
+
+3. If a cluster group exists, navigate to the left **Main Menu** and click on **Cluster Groups**.
```

Comment:
We can combine these two. We should assume a cluster group exists as it's one of the prereqs. But I still think we should include the link in case someone ignore the prereqs and got here. But let's assume they have a cluster group.

Reply:
combined and rearranged.